pkg/core/bootstrap/bootstrap.go

```diff
@@ -106,6 +104,24 @@ func buildRuntime(cfg kuma_cp.Config) (core_runtime.Runtime, error) {
 	return rt, nil
 }
 
+func initializeMetrics(builder *core_runtime.Builder) error {
+	zone := ""
+	switch builder.Config().Mode {
+	case config_core.Remote:
+		zone = builder.Config().Multicluster.Remote.Zone
```

Comment:
I don't know for sure but probably it will look a bit nicer if in Remote `zone` will be equal to 
```go
fmt.Sprintf("Remote[%]", builder.Config().Multicluster.Remote.Zone
```
Because someone might use some generated names for Zones like UUID and it won't be clear that it is actually Remote. What do you think?

Reply:
I think it will be pretty clear that name other than `Standalone` and `Global` are Remote